setup-intent/cs/create-payment-intent.cs

```diff
@@ -0,0 +1,16 @@
+// The sandbox API key passed in 'ApiKeyClientAuthorization' is public.
+// Don't submit any personally identifiable information in any requests made with this key.
+// Sign in to developer.dojo.tech to create your own private sandbox key and use that instead
+// for secure testing.
+
+var paymentIntentsClient = new PaymentIntentsClient(new HttpClient(), new ApiKeyClientAuthorization("sk_sandbox_c8oLGaI__msxsXbpBDpdtwJEz_eIhfQoKHmedqgZPCdBx59zpKZLSk8OPLT0cZolbeuYJSBvzDVVsYvtpo5RkQ"));
+var result = await paymentIntentsClient.CreatePaymentIntentAsync(new CreatePaymentIntentRequest
+{
+    Amount = new()
+    {
+        Value = 1000,
+        CurrencyCode = "GBP"
+    },
+    Reference = "Dojo Cafe | Table no.4.",
+    SetupIntentId = "si_<secret-key>",
```

Comment:
should be consistent with the rest of examples, e.g. below we have `"<Setup intent ID>"`

Reply:
Done.